assets/airports/kdtw.json

```diff
@@ -0,0 +1,4870 @@
+        "POLAR": ["N42.83744102665713", "W83.73541260112334"],
+        "SCUBI": ["N42.68687720617185", "W83.64578247417336"],
+        "TRAML": ["N42.36605287266258", "W83.45626831402373"],
+        "BUF"  : ["N42.92901140779286", "W78.64636230833278"],
+        "MOAWK": ["N42.75030648036240", "W79.84945679071251"],
+        "YQO"  : ["N42.70672684318241", "W80.88784790395742"],
+        "WOLIS": ["N42.62722286565913", "W81.57916260119883"],
+        "ERI"  : ["N42.01740004518572", "W80.29266357780638"],
+        "SPICA": ["N42.50841743783984", "W82.54632568710247"],
+        "RIKKE": ["N42.43940392170810", "W82.74041748397069"],
+        "VERNR": ["N42.40317854534036", "W82.84176636092026"],
+        "WITKR": ["N42.28652069622329", "W83.16522217145581"],
+        "HNN"  : ["N38.75415467606912", "W82.02612305040194"],
+        "RONCO": ["N39.20118519367147", "W82.09671020860260"],
+        "SUBWY": ["N39.69781839613611", "W82.17617798203730"],
+        "MILNR": ["N39.97915462366538", "W82.22167969102009"],
+        "COLTZ": ["N40.49214915564017", "W82.30563354843902"],
+        "CASIO": ["N40.92036166994609", "W82.43069458359089"],
+        "AQQUA": ["N41.25723092685095", "W82.53002930038429"],
+        "WEEDA": ["N41.71591196316977", "W83.06414795270933"],
+        "SVM"  : ["N42.40865424501125", "W83.59387207378454"]
+    },
+    "runways":[
+        {
+            "name": ["3L", "21R"],
+            "end": [
+                ["N42d12m28.20", "W83d21m04.38", "636ft"],
+                ["N42d13m41.85", "W83d20m10.11", "632ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["3R", "21L"],
+            "end": [
+                ["N42d11m44.21", "W83d21m06.49", "633ft"],
+                ["N42d13m10.86", "W83d20m02.65", "632ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["4L", "22R"],
+            "end": [
+                ["N42d12m07.82", "W83d23m02.40", "645ft"],
+                ["N42d13m34.48", "W83d21m58.61", "642ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["4R", "22L"],
+            "end": [
+                ["N42d12m08.37", "W83d22m16.57", "637ft"],
+                ["N42d13m52.37", "W83d20m59.97", "636ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["9L", "27R"],
+            "end": [
+                ["N42d13m01.08", "W83d21m47.40", "639ft"],
+                ["N42d13m03.03", "W83d19m51.71", "635ft"]
+            ],
+            "ils": [false, true]
+        },
+        {
+            "name": ["9R", "27L"],
+            "end": [
+                ["N42d11m56.46", "W83d21m42.22", "636ft"],
+                ["N42d11m58.34", "W83d19m49.33", "629ft"]
+            ],
+            "ils": [false, true]
+        }
+    ],
+    "sids": {
+        "ACO5": {
+            "icao": "ACO5",
+            "name": "Akron Five",
```

Comment:
I used to live here! :smiley: Didn't care for it personally; the place is a bit of a dump.